input/postgres/schema.go

```diff
@@ -182,14 +190,6 @@ func collectSchemaData(ctx context.Context, c *Collection, db *sql.DB, ps state.
 		}
 	}
 
-	if c.GlobalOpts.CollectPostgresFunctions {
```

Comment:
We've discussed that this is probably not useful, but if we're going to stop using it, we should fully remove it.